### DISCUSSION TITLE
The 2nd Human to travel into space?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#695] [Galaxy Trivia] [answered]
First Woman, to Travel into Space?

*Use Name with country flag.

**Comment by peterbaccaro** [comment, marked as answer]
🇺🇸 **Alan Shepard** - U.S. Navy test pilot **Alan Shepard** joined the astronaut program in 1959. He became the first American and the second man in space on May 5, 1961, when he piloted the Mercury spacecraft Freedom 7 on a 490-kilometer (300-mile), 15-minute suborbital flight.

🇷🇺  **Valentina Tereshkova** - The first woman in space, **Valentina Tereshkova**, blazed a trail for the many female spaceflyers who would follow. Tereshkova, a Soviet cosmonaut, was selected from more than 400 applicants to launch on the Vostok 6 mission on June 16, 1963. She was 26 at the time.
